sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -160,19 +158,16 @@ export interface MetricNamespaceName {
 }
 
 // @public (undocumented)
-export class MetricsClient {
+export interface MetricsClientOptions extends PipelineOptions {
+    endpoint?: string;
+}
+
+// @public
+export class MetricsQueryClient {
     constructor(tokenCredential: TokenCredential, options?: MetricsClientOptions);
-    // (undocumented)
     getMetricDefinitions(resourceUri: string, options?: GetMetricDefinitionsOptions): Promise<GetMetricDefinitionsResponse>;
-    // (undocumented)
     getMetricNamespaces(resourceUri: string, options?: GetMetricNamespacesOptions): Promise<GetMetricNamespacesResponse>;
-    // (undocumented)
-    queryMetrics(resourceUri: string, options?: QueryMetricsOptions): Promise<QueryMetricsResponse>;
-}
-
-// @public (undocumented)
-export interface MetricsClientOptions extends PipelineOptions {
-    endpoint?: string;
+    queryMetrics(resourceUri: string, timespan: string, options?: QueryMetricsOptions): Promise<QueryMetricsResponse>;
```

Comment:
It's interesting that we're using `QueryMetricsResponse` - is that just a deserialized response object with the http status code and such? In KeyVault at least we return the result objects by mapping the response to something. Do we have a pattern / preference for either one?

To clarify: I've seen more examples of `queryMetrics(...): Promise<QueryMetricsResult>` or something like that so I was just wondering 👍 Hopefully my question makes sense?

Reply:
I like the 'Result' name you proposed better (my mind was not seeing the different naming on the return).

There are some other clients (communications-chat) that use that convention and it looks good. I'll change them all over.

Reply:
Done!